src/RestApiHelpers/Validation/ValidateActionParametersAttribute.cs

```diff
@@ -20,30 +20,43 @@ public ValidateActionParametersAttribute()
             Order = 1;
         }
 
-        public override void OnActionExecuting(ActionExecutingContext context)
+        public override void OnActionExecuting(ActionExecutingContext executingContext)
         {
-            var descriptor = context.ActionDescriptor as ControllerActionDescriptor;
+            var descriptor = executingContext.ActionDescriptor as ControllerActionDescriptor;
 
             if (descriptor != null)
             {
                 var parameters = descriptor.MethodInfo.GetParameters();
 
                 foreach (var parameter in parameters)
                 {
-                    var argument = context.ActionArguments.ContainsKey(parameter.Name) ?
-                        context.ActionArguments[parameter.Name] : null;
-
-                    EvaluateValidationAttributes(parameter, argument, context.ModelState);
+                    var argument = executingContext.ActionArguments[parameter.Name];
```

Comment:
Did you actually test that this doesn't throw if the collection doesn't contain this key? (I don't know what the runtime type of `ActionArguments` is, but if it's a simple `Dictionary<K, V>`, then it's supposed to throw.)  
If it can throw, the I'd prefer to have a proper check here, even if it shouldn't happen in practice.